content-repo/extra-docs/releases/20.12.1.md

```diff
@@ -0,0 +1,1284 @@
+Cryptocurrency will help classify Cryptocurrency indicators as suspicious when ingested.
+#### Layouts
+**Cryptocurrency Address - Indicator Details**
+
+#### Scripts
+##### CryptoCurrenciesFormat
+Verifies that a crypto address is valid and only returns the address if it is valid.
+
+
+---
+
+### New: Darktrace Pack v1.0.0 (Partner Supported)
+#### Classifiers
+##### Darktrace Incident Classifier
+Classifies all incidents fetched by the Darktrace integration as "Darktrace Model Breach" type incidents for purposes of using the Integration Playbooks.
+##### Darktrace Model Breach Mapper
+Maps model breach fields to mappings for use in Integration Playbooks.
+#### Incident Fields
+- **Darktrace Comment Count**
+Number of comments on the Darktrace model breach
+- **Darktrace Device Hostname**
+Hostname associated with device in Darktrace
+- **Darktrace Device ID**
+
+- **Darktrace Device IP**
+IP address associated with device in Darktrace
+- **Darktrace Device Label**
+Device label associated with device in Darktrace
+- **Darktrace Device MAC**
+MAC address associated with device in Darktrace
+- **Darktrace Device Vendor**
+Vendor associated with device in Darktrace
+- **Darktrace Model Breach Description**
+Description of the Model Breach
+- **Darktrace Model Breach ID**
+Darktrace Model Breach ID (PBID) of the breached model
+- **Darktrace Model Breach Score**
+Darktrace Model Breach Score
+- **Darktrace Model ID**
+Model ID (PID) associated with the breached model in Darktrace
+- **Darktrace Model Name**
+Name associated with the breached model in Darktrace
+- **Darktrace Model Priority**
+Priority of the breached model in Darktrace
+- **Darktrace Model Tags**
+Tags associated with the breached Darktrace model
+- **Darktrace Model UUID**
+Model UUID associated with the breached model in Darktrace
+#### Incident Types
+- **Darktrace Model Breach**
+
+#### Integrations
+##### Darktrace
+Rapid detection of malicious behavior can make all the difference in the response to a security event. This pack includes configurations to combine the world-class threat detection of Darktrace with the synchrony and automation abilities of XSOAR, allowing security teams to investigate and manage security events before they have time to escalate.
+
+#### Playbooks
+##### Handle Darktrace Model Breach
+Handles each fetched Darktrace model breach by gathering additional detail about the activity and device, providing enrichment data from Darktrace and XSOAR, linking similar incidents, and giving the ability to acknowledge the model breach and close the incident.
+
+
+---
+
+### New: ModulesManagement Pack v1.0.0
+#### Scripts
+##### GetInstances
+Returns integration instances configured in Cortex XSOAR. You can filter by instance status and/or brand name (vendor).
+
+
+---
+
+### New: Public DNS Feed Pack v1.0.0
+#### Integrations
+##### Public DNS Feed
+Fetches IP addresses from the Public DNS feed.
+
+
+---
+
+### New: Rapid Breach Response Pack v1.0.0
```

Comment:
@glicht @anara123 do we want to highlight this one somehow in these release notes?

Reply:
Yes, we do. Maybe you can move it to the top